libqtile/core/manager.py

```diff
@@ -1224,6 +1224,61 @@ def cmd_spawn(self, cmd):
             os.close(r)
             return int(pid)
 
+    def cmd_spawn_terminal(self, cmd=[]):
+        """Try to guess terminal or read default from config and open empty or
+        spawn command
+
+        cmd may be a string, which is parsed by shlex.split, or a list (similar
+        to subprocess.Popen).
+        """
+        if isinstance(cmd, str):
+            args = shlex.split(cmd)
+        else:
+            args = list(cmd)
+
+        test_terminals = [
+                ['x-terminal-emulator', '-?'],
+                ['xterm', '-help'],
+                ['urxvt', '-help'],
+                ['st', '-v'],
+                ['gnome-terminal', '-?'],
+                ['terminator', '-v'],
+                ['lxterminal', '-v'],
+                ['mate-terminal', '-h'],
+                ['roxterm', '-?'],
+                ['tilda', '--help'],
+                ['guake', '-h'],
+                ['eterm', '-h'],
+                ['sakura', '-v'],
+                ['konsole', '--help'],
+                ['hyper', '-v']
+                ['alacritty', '-V'],
+                ]
+
+        default_terminal = getattr(self.config, "default_terminal", None)
+        if default_terminal:
+            full_cmd = [default_terminal]
+            if len(args):
+                full_cmd += ['-e'] + args
+            return self.cmd_spawn(full_cmd)
+
+        to_null = ['>', '/dev/null']
+        for terminal in test_terminals:
```

Comment:
Why not use get_default_terminal here?

Reply:
sorry, forgot to remove this command - it is not needed anymore. Will add something to documentation